sections/01_abstract.md

```diff
@@ -3,3 +3,14 @@
 Offspring of the [Deep Review](https://github.com/greenelab/deep-review).
 We cite the Deep Review like [@doi:10.1101/142760] or [@tag:deep_review].
 The source repository is [@url:https://github.com/greenelab/manubot-rootstock].
+
+Equations can be integrated and labeled like this `{{ '{{ \'{#eq:label}\' }}' }}` and then
+referenced using `{@eq:label}` like this for Equation @eq:line.
+
+$$ y = mx + b $$ {{ '{#eq:line}' }}
+
+Images can be included and labeled like `{{ '{{ \'{#fig:label}\' }}' }}` and then
```

Comment:
It'd be nice if we can find a way with `jinja`. Based on my testing earlier today, adding a space (e.g. `{ #eq:line}`) did not work for equation labels (I didn't test here with a space for figure labels). I saw `comment_start_string` in the documentation but wasn't sure if (or how) we'd like to implement them.